code/modules/mob/living/carbon/human/species_types/angel.dm

```diff
@@ -142,4 +142,4 @@
 		H.movement_type &= ~FLYING
 		override_float = FALSE
 		H.pass_flags &= ~PASSTABLE
-		H.CloseWings()
\ No newline at end of file
+		H.CloseWings()

```

Comment:
huh

Reply:
The `EditorConfig` made me put a newline at the end